server/public/styles.css

```diff
@@ -0,0 +1,4 @@
+.push-right{
+position: relative;
+left: 1.2rem;
+}
```

Comment:
You probably meant to add left margin here. Bootstrap has spacing helpers.

https://getbootstrap.com/docs/4.4/utilities/spacing/#notation